spec/factories/file_versions_factory.rb

```diff
@@ -5,5 +5,10 @@
     association :repository
     commit_sha { Faker::Crypto.sha1 }
     path { generate(:filepath) }
+
+    after(:create) do |file_version|
```

Comment:
Can you add a todo comment that we want to run this asyncronously later on?

Reply:
That's for the backend to do ;)